components/table/ReportTable.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { Flex } from "@chakra-ui/layout"
+import { TimesheetEntry } from "@/lib/types/apiTypes"
+
+interface ReportTableProps {
+    entries: TimesheetEntry[]
+}
+
+function ReportTable({ entries }: ReportTableProps): JSX.Element {
+    const responseLength = entries.length
+    const totalQuantity = entries.reduce(
+        (total, currentItem) => (total += currentItem.quantity),
+        0
+    )
+
+    return (
+        <Flex
+            backgroundColor="white"
+            border="solid 0.5px"
+            borderColor="gray.400"
+            borderRadius="0.2rem"
+        >
+            {totalQuantity > 0 ? (
```

Comment:
This component is going to get somewhat complicated in the long run, so it might be better to render
```typescript
            {totalQuantity > 0 ? (
                            <>
                    The total number of hours made between {entries[0].date} and{" "}
                    {entries[responseLength - 1].date} is {totalQuantity}
                </>
            ) : (
                <p>
                    No hours between {entries[0].date} and{" "}
                    {entries[responseLength - 1].date}
                </p>
            )}
```
as a separate component (something like `TotalHours`). I think suitable props would be `totalQuantity`, `startDate` and `endDate`, with the dates non-nullable. Then only render the component if `startDate` and `endDate` are defined.